Проект по 2-ому спринту, версия 1

yareliance opened the pull request:
Спринт 2

Commit:
Проект по 2-ому спринту, версия 1

alesyaisachenko commented on src/main/java/main.java:
```diff
@@ -0,0 +1,20 @@
+import model.constants.*; //импорт компанентов подпакета
+import service.ShoppingCart;
+
+public class main {
```
Обязательно доработать:
 main -- нужно  Main, классы именуются в таком формате

alesyaisachenko commented on src/main/java/main.java:
```diff
@@ -0,0 +1,20 @@
+        Food[] products = {meat, redApples, greenApples};
+        ShoppingCart cart = new ShoppingCart(products); //Объект корзина
+
+        System.out.println("Общая сумма товаров без скидки: " + cart.getSumWithoutDiscount());
```
Можно лучше:
для сборки строк лучше использовать String.format() или System.out.printf

alesyaisachenko commented on src/main/java/model/Apple.java:
```diff
@@ -0,0 +1,26 @@
+
+
+public class Apple extends Food{
+    public String colour;  // у яблок есть дополнительное поле - цвет яблок
```
Это нужно доработать:
public -- нужно выбрать другой модификатор доступа

Commit:
Проект по 2-ому спринту, версия 2:
1. Изменено название класса Main.
2. Изменен уровень доступа к полю цвета яблок
3. Изменен вывод сум и добавлено форматирование.
